config.libsonnet

```
@@ -1,7 +1,7 @@
 {
   _config+:: {
     //any config variables should go here
-
+    bucketErrorStateAlertTime: '2m',
```

We push every 5m, so 2m is just 1 report.
Let's change both bucket and State to 6m (two cycles)
```suggestion
    bucketErrorStateAlertTime: '6m',
```